component-css/input.css

```diff
@@ -263,10 +44,12 @@
 .content-box .input-component-container .input-container{
     padding: 1rem;
     margin: 1rem 0rem 1rem 0rem;
-    background-color: transparent;
+    background-color: #E7E8E8;
```

Comment:
Also, this color code has been used a lot at multiple places, I suggest to make this too as a variable.